tests/conftest.py

```diff
@@ -31,3 +32,15 @@ def env(request):
     yield env
 
     env.close()
+
+
+@pytest.fixture(scope="session")
```

Comment:
`scope="module"` for now, since we don't expect to have more than a couple of modules of tests for this env type.

Reply:
Done